SwiftyJSON.xcodeproj/project.pbxproj

```diff
@@ -672,7 +684,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
```

Comment:
What is the reason for changing this and why isn't it mentioned in the pull request?